tests/test_meshtastic_utils.py

```
@@ -212,6 +213,130 @@ def test_on_meshtastic_message_no_text(self):
+            "mmrelay.meshtastic_utils._submit_coro"
+        ) as mock_submit_coro, patch(
+            "mmrelay.meshtastic_utils.logger"
+        ):
+            mock_interface = MagicMock()
+            mock_interface.myInfo.my_node_num = 999
+
+            on_meshtastic_message(reaction_packet, mock_interface)
+
+            mock_submit_coro.assert_called_once()
+            # Ensure we scheduled the matrix relay coroutine
+            scheduled_coro = mock_submit_coro.call_args.args[0]
```

Ruff (0.14.6)

Local variable `scheduled_coro` is assigned to but never used

Remove assignment to unused variable `scheduled_coro`

(F841)